concepticondata/conceptlists/Gabelentz-1892-1074.tsv

```diff
@@ -0,0 +1,1075 @@
+Gabelentz-1892-1074-819	819	Leugnen	to deny	VIII. Verba	A. Sagen, sprechen u.s.w.	755	DENY
+Gabelentz-1892-1074-820	820	Verweigern	to refuse	VIII. Verba	A. Sagen, sprechen u.s.w.	49	REFUSE
+Gabelentz-1892-1074-821	821	Streiten	to argue, to quarrel	VIII. Verba	A. Sagen, sprechen u.s.w.	3302	QUARREL (VERB)
+Gabelentz-1892-1074-822	822	Lügen	to lie	VIII. Verba	A. Sagen, sprechen u.s.w.	601	LIE (MISLEAD)
+Gabelentz-1892-1074-823	823	Schelten, schimpfen	to scold	VIII. Verba	A. Sagen, sprechen u.s.w.	299	SCOLD
+Gabelentz-1892-1074-824	824	Betrügen, täuschen	to deceive	VIII. Verba	A. Sagen, sprechen u.s.w.	3230	DECEIVE
+Gabelentz-1892-1074-825	825	Denken	to think	VIII. Verba	B. Denken u.s.w.	2271	THINK
+Gabelentz-1892-1074-826	826	Gedenken	to commemorate	VIII. Verba	B. Denken u.s.w.		
+Gabelentz-1892-1074-827	827	Träumen	to dream	VIII. Verba	B. Denken u.s.w.	1920	DREAM (SOMETHING)
+Gabelentz-1892-1074-828	828	Kennen	to know someone	VIII. Verba	B. Denken u.s.w.	2248	KNOW (SOMEBODY)
+Gabelentz-1892-1074-829	829	Wissen	to know something	VIII. Verba	B. Denken u.s.w.	1410	KNOW (SOMETHING)
+Gabelentz-1892-1074-830	830	Können	to be able	VIII. Verba	B. Denken u.s.w.	972	BE ABLE
+Gabelentz-1892-1074-831	831	Finden	to find	VIII. Verba	B. Denken u.s.w.	719	FIND
+Gabelentz-1892-1074-832	832	Messen	to measure	VIII. Verba	B. Denken u.s.w.	1506	MEASURE
+Gabelentz-1892-1074-833	833	Vergleichen	to compare	VIII. Verba	B. Denken u.s.w.	3429	COMPARE
+Gabelentz-1892-1074-834	834	Glauben	to believe	VIII. Verba	B. Denken u.s.w.	1890	BELIEVE
+Gabelentz-1892-1074-835	835	Lieben	to love	VIII. Verba	B. Denken u.s.w.	923	LOVE
+Gabelentz-1892-1074-836	836	Hoffen	to hope	VIII. Verba	B. Denken u.s.w.	1092	HOPE
+Gabelentz-1892-1074-837	837	Wollen	to want	VIII. Verba	B. Denken u.s.w.	1784	WANT
+Gabelentz-1892-1074-838	838	Wählen	to choose	VIII. Verba	B. Denken u.s.w.	1750	CHOOSE
+Gabelentz-1892-1074-839	839	Wünschen, begehren	to wish for	VIII. Verba	B. Denken u.s.w.	2117	DESIRE
+Gabelentz-1892-1074-840	840	Warten	to wait	VIII. Verba	B. Denken u.s.w.	1590	WAIT (FOR)
+Gabelentz-1892-1074-841	841	Lauern	to lurk	VIII. Verba	B. Denken u.s.w.		
+Gabelentz-1892-1074-842	842	Aufpassen	to pay attention	VIII. Verba	B. Denken u.s.w.		
+Gabelentz-1892-1074-843	843	Versuchen	to try	VIII. Verba	B. Denken u.s.w.	1812	TRY
+Gabelentz-1892-1074-844	844	Fürchten	to fear	VIII. Verba	B. Denken u.s.w.	1419	FEAR (BE AFRAID)
+Gabelentz-1892-1074-845	845	Sich wundern	to wonder	VIII. Verba	B. Denken u.s.w.		
+Gabelentz-1892-1074-846	846	Sich schämen	to be ashamed of oneself	VIII. Verba	B. Denken u.s.w.	1894	ASHAMED
+Gabelentz-1892-1074-847	847	Ehren	to honour	VIII. Verba	B. Denken u.s.w.	3628	HONOR (SOMEBODY)
+Gabelentz-1892-1074-848	848	Erschrecken	to be scared or to scare someone	VIII. Verba	B. Denken u.s.w.	2109	FEAR OR FRIGHTEN
+Gabelentz-1892-1074-849	849	Sich freuen	to be happy	VIII. Verba	B. Denken u.s.w.	1495	HAPPY
```

Comment:
Likewise, I'm not sure it can be mapped to a property concept set.

Reply:
I'd leave it as is.